bodhi/client/__init__.py

```diff
@@ -368,33 +381,44 @@ def query_buildroot_overrides(url, user=None, **kwargs):
 
 
 @overrides.command('save')
-@click.argument('nvr')
-@click.option('--duration', default=7, type=click.INT,
-              help='Number of days the override should exist.')
-@click.option('--notes', default="No explanation given...",
-              help='Notes on why this override is in place.')
-@click.option('--user', envvar='USERNAME')
-@click.option('--password', prompt=True, hide_input=True)
-@click.option('--staging', help='Use the staging bodhi instance',
-              is_flag=True, default=False)
+@add_options(save_edit_options)
 @url_option
-def save_buildroot_overrides(nvr, duration, notes, user, password, staging, url):
+def save_buildroot_overrides(user, password, url, staging, **kwargs):
     """
-    Create or modify a buildroot override.
+    Create a buildroot override.
 
     Args:
-        nvr (unicode): The NVR of the update you wish to submit as a buildroot override.
-        duration (int): The number of days you wish the override to be active for.
-        notes (unicode): Notes to leave on the override.
         user (unicode): The username to authenticate as.
         password (unicode): The user's password.
+        url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
+                       True.
         staging (bool): Whether to use the staging server or not.
+        kwargs (dict): Other keyword arguments passed to us by click.
+    """
+    client = bindings.BodhiClient(base_url=url, username=user, password=password, staging=staging)
+    resp = client.save_override(**kwargs)
+    print_resp(resp, client)
+
+
+@overrides.command('edit')
+@add_options(save_edit_options)
+@click.option('--expired', help='Expire the override', is_flag=True, default=False)
+@url_option
+def edit_buildroot_overrides(user, password, url, staging, **kwargs):
+    """
+    Edit a buildroot override.
+
+    Args:
+        user (unicode): The username to authenticate as.
+        password (unicode): The user's password.
         url (unicode): The URL of a Bodhi server to create the update on. Ignored if staging is
                        True.
+        staging (bool): Whether to use the staging server or not.
         kwargs (dict): Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, username=user, password=password, staging=staging)
-    resp = client.save_override(nvr=nvr, duration=duration, notes=notes)
+    kwargs['edited'] = kwargs['nvr']
+    resp = client.save_override(**kwargs)
```

Comment:
It seems that the only difference between save and edit is the ```kwargs['edited']``` line. What if we make a helper function that accepts a boolean called ```edit``` and then we can make the code just do

```
if edit:
    kwargs['edited'] = kwargs['nvr']
```

Then the save and edit function here can both share the other code.